data/human/human missions.txt

```diff
@@ -7453,3 +7453,775 @@ mission "Adelita 5"
+			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you 3 million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`
+			label questions
+			choice
+				`	"If I find other stones, can you harmonize them as well?"`
+					goto harmonize
+				`	"What will you do now?"`
+					goto next
+				`	"Why don't you just leave? I could take you off world."`
+					goto offworld
+				`	"Thanks for all you've done. I hope we meet again."`
+				`	"So long."`
+			label bye
+			`	"I appreciate you comin' down here, and standin' for somethin'. I always thought there was somethin' right uncommon 'bout you. And you proved in spades. Take care now, and hopefully we'll meet again."`
+				decline
+			label next
+			`	The injured man gets a distant look in eyes, "Well, I figure I'll just get back to doin' what I am doin'. Lots of things to figure, lots of cords to sell. Always work, and excitement for someone in my boots. I wager one of these days I'll shake loose of my masters. But until then I'll abide."`
+				goto questions
+			label harmonize
+			`	Weland looks at you wistfully, and then, with a tinge of sadness, shakes his head. "I wager it's a sight too dangerous for that. Each stone is grown, or sung differently, it would take me months to figure out how to harmonize another one. And in the meantime, I am still watched, still unable to leave, still the property of others. If you left somethin' that valuable with me again, they'd take it, and this time to somewhere you wouldn't find it."`
+			`	He shrugs, "Perhaps, some other time. On a different world, in different circumstances. But, not now, not here, and not like this."`
+				goto questions
+			label offworld
+			`	He gives you a patronizing smile. "Folks that did this to me. They've been doin' this kind of work for a while, and I, well, I am like the crown jewel of their collection. They have hooks into me, inside of me, all around me, that prevent me from leaving, at least alive. They like to give me the illusion of space, pretend they are... gentile 'bout the whole thing. But make no mistake, I am not free, and I cannot go."`
+			`	His face briefly flashes with mirth, "For now, that is."`
+				goto questions
+
+mission "Stone Harmonization 3surtr"
+	name "Board the Aotrou"
+	description "Find the Aotrou, disable it, board it, and retrieve the Blood of Surtr."
+	source "Wayfarer"
+	waypoint "Naper"
+	destination "Wayfarer"
+	to offer
+		has "Stone Harmonization 2surtr: done"
+	on offer
+		conversation
+			`Weland is waiting for you outside a nearly empty eatery. He turns his wheelchair to face you as you approach.`
+			`	"Well, as you may have surmised," he nods toward his covered feet, that were until recently, chained, "I am someone else's property."`
```

Comment:
```suggestion
			`	"Well, as you may have surmised," he nods toward his covered feet, that were, until recently, chained, "I am someone else's property."`
```
Alternatively (because this is a lot of commas):
```suggestion
			`	"Well, as you may have surmised," he nods toward his covered feet that were until recently chained, "I am someone else's property."`
```